src/main/java/com/hrznstudio/titanium/block/tile/fluid/SidedFluidTank.java

```diff
@@ -71,6 +79,22 @@ public Rectangle getRectangle() {
         return new Rectangle(this.getPosX(), this.getPosY(), 18 - 1, 46 - 1);
     }
 
+    public SidedFluidTank setButtonCoords(int buttonX, int buttonY) {
```

Comment:
Make this method part of the IFacingHandler interface and call it `setFacingHandlerPos`